lib/import_pipelines.js

```diff
@@ -149,9 +154,38 @@ function createPeliasElasticsearchPipeline(){
   return entryPoint;
 }
 
+/*
+ * Create a single stream from many CSV files
+ */
+function createFullRecordStream(files) {
```

Comment:
this file officially feels too big and should be broken up into separate stream factories